Rework backup (#8406)
kyma-project/kyma: docs/kyma/08-05-backup-with-velero.md

          --wait
      ```

      >**NOTE:** For details on configuring and installing Velero on Azure, read more about the [Velero plugin for Azure](https://github.com/vmware-tanzu/velero-plugin-for-microsoft-azure/blob/master/README.md) document.

>       >**CAUTION:** If you are using AKS, set the **AZURE_RESOURCE_GROUP** to the name of the auto-generated resource group. This resource group is created when you provision your cluster on Azure and contains your cluster's virtual machines/disks.

Reviewer: ```suggestion
      >**CAUTION:** If you use AKS, set the **AZURE_RESOURCE_GROUP** to the name of the auto-generated resource group. This resource group is created when you provision your cluster on Azure. It contains your cluster's virtual machines/disks.
```
